scripts/alias_index.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
+
+# This script will get all the ElasticSearch indexes that uses a given alias, the alias will
+# be removed from those indexes, before the index name that is passed into this script
+# is aliased using the given alias.
+# e.g. sh ./scripts/alias_index.sh localhost 9200 bi-dev-01082018 bi-dev
+# - The indexes aliased to 'bi-dev' will have their aliases removed (if any exist).
+# - The passed in index name will be aliased using the given alias ('bi-dev').
+# https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-aliases.html#indices-aliases
+
+SCRIPT_NAME=$0
+REQUIRED_NUM_ARGS=4
+
+# Fail fast if we get any errors
+set -e
```

Comment:
Other helpful flags

```
# Exit on error inside any functions or subshells.
set -o errtrace
# Do not allow use of undefined vars. Use ${VAR:-} to use an undefined VAR
set -o nounset
# Feed error up to top. For example catch the error in case mysqldump fails (but gzip succeeds) in `mysqldump |gzip`
set -o pipefail
# Turn on traces, useful while debugging but commented out by default
# set -o xtrace
```